docs/source/daml/intro/7_Composing.rst

```diff
@@ -78,6 +78,13 @@ import only the selected names:
 
   import DA.List (sortOn, groupOn)
 
+If your module contains any DAML Scripts, you need to import the
+corresponding functionality:
+
+.. code-block:: daml
+
+  import Daml.Script
```

Comment:
Should this also mention `ApplicativeDo`?

Reply:
Good idea, I’ll add something.

Reply:
We don’t actually use it in this example after the changes. It is mentioned in a preceding section so I’m omitting it here.